[web] add cdn warning (#185)
Scandit/data-capture-documentation: docs/sdks/web/add-sdk.md

## Install via CDN

:::warning Important considerations when using CDNs

> While CDNs offer a convenient way to get started, they introduce significant dependencies into your application. Your app's functionality becomes directly tied to the CDN's availability and performance. Any CDN outages or slowdowns will immediately affect your users' experience.

Author: ```suggestion
CDNs offer a convenient way to get started but they introduce dependencies into your application. Your app's functionality becomes directly tied to the CDN's availability and performance. Any CDN outages or slowdowns will immediately affect your users' experience.
```

For production environments, we recommend:

> 1. **Self-hosting** the SDK files on your own infrastructure as the preferred option, where you can:

Reviewer: ```suggestion
1. **Self-hosting** the SDK files on your own infrastructure, where you are strongly encouraged to:
```

   - Configure optimal cache headers and compression settings
   - Set correct MIME types for .wasm, .js and .model files
   - Control Content-Length headers for accurate loading progress
   - Minimize request redirections and network latency
   - Implement your own fallback mechanisms

> 2. If self-hosting isn't feasible, using a **paid enterprise CDN service** that provides:

Author: ```suggestion
2. If self-hosting isn't feasible, use a **paid enterprise CDN service** that provides:
```
